lib/Driver/Driver.cpp

```diff
@@ -2031,6 +2031,12 @@ const ToolChain *Driver::getToolChain(const ArgList &Args) const {
       TC = new toolchains::Darwin(*this, Target);
       break;
     case llvm::Triple::Linux:
+      if (Target.isAndroid()) {
```

Comment:
(Maybe it is a Unix-like target too, and then not a Unix-like OS. But that's not about the Driver.)